src/models/language/nativeLanguageCollection.ts

```diff
@@ -3,6 +3,7 @@ import { ILanguage } from './language';
 export interface INativeLanguageCollection {
   bat: ILanguage;
   c: ILanguage;
+  ceylon: ILanguage;
```

Comment:
If in the end there's no extension supporting this language this line will have to be removed.

Reply:
https://github.com/bjansen/vs.language.ceylon/blob/master/package.json#L16